google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -101,6 +101,21 @@ public BucketInfo apply(com.google.api.services.storage.model.Bucket pb) {
   private final String locationType;
   private final Logging logging;
 
+  public enum PublicAccessPrevention {
+    ENFORCED("enforced"),
+    UNSPECIFIED("unspecified");
```

Comment:
Is one of these two values the "default" value? Can we add a comment calling that out?